docs/howto/release-git-lfs.md

```diff
@@ -0,0 +1,340 @@
+     Merge: 3f3faa90 a55b7fd9
+     Author: Taylor Blau <ttaylorr@github.com>
+     Date:   Thu Jul 26 14:48:39 2018 -0500
+
+         Merge pull request #3150 from git-lfs/release-next
+
+             release: v2.n.0
+     $ git tag -s v2.n.m
+     $ git describe HEAD
+     v2.n.m
+     ```
+
+  4. Begin building release artifacts.
+
+     * To build `*.tar.gz` artifacts, run `make release`.
+
+     * To build `*.zip` artifacts, run `make release` (as above), and follow the
+       additional instructions in the section ["For Windows"](#for-windows)
+       below.
+
+     * To build `*.deb` and `*.rpm` artifacts, follow the additional
+       instructions in ["For Linux"](#for-linux) below.
+
+  5. Create a GitHub release for the new version number, including the
+     changelog and the standard copy materials, as below:
+
+     ```
+     <CHANGELOG.md additions>
+
+     ## Packages
+
+     Up to date packages are available on
+     [PackageCloud](https://packagecloud.io/github/git-lfs) and
+     [Homebrew](http://brew.sh/).
+
+     [RPM RHEL 6/CentOS 6](https://packagecloud.io/github/git-lfs/packages/el/6/...)
+     [RPM RHEL 7/CentOS 7](https://packagecloud.io/github/git-lfs/packages/el/7/...)
+     [Debian 7](https://packagecloud.io/github/git-lfs/packages/debian/wheezy/...)
+     [Debian 8](https://packagecloud.io/github/git-lfs/packages/debian/jessie/...)
+     [Debian 9](https://packagecloud.io/github/git-lfs/packages/debian/stretch/...)
+
+     ## SHA-256 hashes:
+
+     **</path/to/artifact>**
+     <artifact SHA-256>
+     ```
+
+  6. Upload the `*.tar.gz`, `*.zip` and Windows Installer assets to the release
+     notes, and rename them according to the following list:
+
+     * Darwin 386
+     * Darwin AMD64
+     * FreeBSD 386
+     * FreeBSD AMD64
+     * Linux 386
+     * Linux AMD64
+     * Windows Installer
+     * Windows 386
+     * Windows AMD64
+
+     GitHub does not allow you (at the time of writing) to edit the name of a
+     release asset from the website interface. Instead, list all release
+     artifacts by:
+
+     ```ShellSession
+     $ curl -n https://api.github.com/repos/git-lfs/git-lfs/releases |
+       jq '.[0].assets | .[] | {"id","name","label"} '
+     ```
+
+     And update a single release asset's name by running:
+
+     ```ShellSession
+     $ curl -XPATCH -in \
+       https://api.github.com/repos/git-lfs/git-lfs/releases/assets/8712424 \
+       -d '{
+         "name": "git-lfs-linux-amd64-v2.5.2.tar.gz",
+         "label": "Linux AMD64"
+       }'
+     ```
+
+     Make sure to also include the SHA-256 signatures of all of the artifact
+     hashes, by running:
+
+     ```ShellSession
+     $ pwd
+     /go/src/github.com/git-lfs/git-lfs/bin/releases
+     $ shasum -a256 * | awk '{ print "**" $2 "**\n" $1 "\n" }'
+     ```
+
+  7. Push the tag, via:
+
+     ```ShellSession
+     $ git push origin v2.n.m
+     ```
+
+     And publish the release on GitHub.
+
+	8. Update the `_config.yml` file in
```

Comment:
It looks like this step is improperly indented, and that's also causing it to be treated similarly to a code-block when the markdown is rendered.